labelCloud/model/point_cloud.py

```diff
@@ -259,5 +324,11 @@ def print_details(self) -> None:
         print_column(["Point Cloud Minimums:", np.round(self.pcd_mins, 2)])
         print_column(["Point Cloud Maximums:", np.round(self.pcd_maxs, 2)])
         print_column(
-            ["Initial Translation:", np.round(self.init_translation, 2)], last=True
+            ["Initial Translation:", np.round(self.init_translation, 2)],
         )
+        if self.SEGMENTATION:
```

Comment:
Was that for debugging or should it stay?

Reply:
I think some stats would be beneficial for the annotators and reviewers but probably more useful to have it in the GUI. Remove the code for now here. [d42492b](https://github.com/ch-sa/labelCloud/pull/97/commits/d42492bf37bbb8570391619ad1a329dc1608eb38)